lib/Config/Menu.php

```diff
@@ -371,6 +372,16 @@ public function setup() {
       'mailpoet-experimental',
       [$this, 'experimentalFeatures']
     );
+
+    // display loggs page
+    $this->wp->addSubmenuPage(
+      true,
+      $this->setPageTitle('Logs'),
+      '',
+      AccessControl::PERMISSION_ACCESS_PLUGIN_ADMIN,
+      'logs',
```

Comment:
The url slug in spec is `mailpoet-logs`. I think it would be better to use the one from spec since `http://example.com/wp-admin/admin.php?page=logs` may cause some conflicts.